joark/src/main/kotlin/no/nav/helsearbeidsgiver/inntektsmelding/joark/JournalfoerImRiver.kt

```diff
@@ -164,14 +152,14 @@ class JournalfoerImRiver(
             Metrics.dokArkivRequest.recordTime(dokArkivClient::opprettOgFerdigstillJournalpost) {
                 dokArkivClient.opprettOgFerdigstillJournalpost(
                     tittel = "Inntektsmelding",
-                    gjelderPerson = GjelderPerson(inntektsmelding.identitetsnummer),
+                    gjelderPerson = GjelderPerson(inntektsmelding.sykmeldt.fnr.verdi),
```

Comment:
Er det sånn at dette er ditt synspunkt på `verdi` og `toString()` for Orgnr og Fnr, siden du konsekvent bruker verdi? Jeg har brukt toString en del, men gidder ikke det hvis følgende meme er konsensus i teamet 😄
![image](https://github.com/user-attachments/assets/eae1108e-53a6-4c9c-bf69-4edbb67271b9)

Reply:
Memen treffer blink.